go/api/base_client.go

```diff
@@ -540,3 +541,17 @@ func (client *baseClient) PingWithMessage(message string) (string, error) {
 	}
 	return response.Value(), nil
 }
+
+func (client *baseClient) Del(keys []string) (int64, error) {
```

Comment:
Please return int64 wrapped in a Result as `Result[int64]` in order to pass the information regarding `nil int64` in case of an error. Please refer to other command implementations.